src/object.js

```diff
@@ -195,12 +242,13 @@ ReflectionObject.prototype.setParsedOption = function setParsedOption(name, valu
         });
         if (opt) {
             // If we found an existing option - just merge the property value
+            // (If it's a feature, will just write over)
             var newValue = opt[name];
-            util.setProperty(newValue, propName, value);
+            util.setProperty(newValue, propName, value, isFeature);
```

Comment:
Name is just `features`. Why do you think the regex is wrong?

Reply:
typo, the regex is only wrong if `name` is the fully feature path.  If it's always just `features`, this will overwrite the entire features object for each specified feature right?  It'll only keep the last one?